### ISSUE TITLE
Survey status chip

### ISSUE THREAD

**Issue by rebecarubio** [zetkin/app.zetkin.org#987]
## Description
A user should be able to know the status of the survey

## Relevant Job Stories
When a survey is no longer relevant, I want to close the survey, so that no more people can respond to it

## Prerequisites
Don't start until the pages have been created in #982 

## Requirements
- [x] Add status chip in survey header 
- [x] Add status chips objects : open, active, scheduled, draft and closed.

## Possible implementations
Mimic the patterns used in call assignments (`CallAssignmentStatusChip.tsx`).

## Design specifications
Figma --> https://www.figma.com/file/EjLcpjKWgNizCb5PRBF51P/Zetkin-3?node-id=2517%3A36483&t=Zm38Y7wFONKOsZfv-0

Open survey status
![chipppp](https://user-images.githubusercontent.com/36491300/217789203-737bf4e0-862f-485a-bdb3-836fc6e30863.png)

List of survey status (Active is not relevant for surveys)
![chipppppppps](https://user-images.githubusercontent.com/36491300/217805458-2f1272b9-1e10-41a0-8058-c8a8c49cfbd7.png)
